analyzer/java/com/engflow/bazel/invocation/analyzer/bazelprofile/BazelProfile.java

```diff
@@ -125,8 +125,7 @@ private BazelProfile(JsonObject profile) {
               // TODO: Use success response to take action on errant events.
               profileThread.addEvent(object);
             });
-    if (!threads.values().stream()
-        .anyMatch(t -> BazelProfileConstants.THREAD_MAIN.equals(t.getName()))) {
+    if (!threads.values().stream().anyMatch(BazelProfileConstants::isMainThread)) {
```

Comment:
should we move this chain to a meaningful method? For example, I find this more readable:

```suggestion
    if (!containsMainThread()) {
```

by defining:

```java
private boolean containsMainThread() {
    return threads.values().stream().anyMatch(BazelProfileConstants::isMainThread)
}
```

Reply:
I like it, done!